addons/l10n_in_edi_ewaybill/models/account_move.py

```diff
@@ -107,7 +107,16 @@ def l10n_in_edi_ewaybill_send(self):
                 raise UserError(_("You can only create E-waybill from posted invoice"))
             errors = edi_format._check_move_configuration(move)
             if errors:
-                raise UserError(_("Invalid invoice configuration:\n\n%s", '\n'.join(errors)))
+                invoice_line = self.invoice_line_ids.filtered(lambda line: (not line.l10n_in_hsn_code or not line.tax_ids))
```

Comment:
This returns many other errors that don't have anything to do with the HSN or the taxes set. (partner, reference, product type,...)

Reply:
Yes here many things will check and display all errors at once, but we do have not any other option for displaying those lines.
we can do this - if there are lines with empty HSN or tax then we will raise RedirectWarning otherwise UserError 

but this solution is not convenient so after discussing with the Indian PO, we finalized that always raise RedirectWarning so user can open a list of lines which not have  HSN or tax.